pkg/controllers/proxy/network_services_controller.go

```diff
@@ -225,6 +225,15 @@ type endpointSliceInfo struct {
 // map of all endpoints, with unique service id(namespace name, service name, port) as key
 type endpointSliceInfoMap map[string][]endpointSliceInfo
 
+func checkRpFilter1(ifname string) bool {
```

Comment:
nitpic, but this would probably be better named `checkRPFilter1` since `RP` is an acronym for reverse path.